packages/nibijs/src/query/query.ts

```diff
@@ -3,7 +3,7 @@ import {
   DistributionExtension,
   GovExtension,
   IbcExtension,
-  QueryClient,
+  QueryClient as StargateQueryClient,
```

Comment:
Is it necessary to alias here?

Reply:
It's confusing in the API docs because it doesn't come with a hyperlink

Reply:
Since it's really `QueryClient` (from `cosmjs/stargate`), it's more clear to call it `StargateQueryClient`. It's basically the cousin of `SigningStargateClient`